packages/themes/README.md

```diff
@@ -1,3 +1,33 @@
-# Themes
+# Designsystem Themes
 
-This library contains themes for Designsystemet.
+This package provides prebuilt themes for use with Designsystemet at KS Digital. Themes are created using the [Theme Builder from Designsystemet](https://theme.designsystemet.no/)
+
+## Supported Themes
+
+The following themes are officially supported:
+
+- **Ledsagerbevis**
+
+Additionally, a base style is provided and should be applied first. We aim to keep this base style as minimal as possible.
+
+## Usage
+
+To use the themes, include the base styles and one of the supported themes in your CSS or JavaScript/TypeScript files as shown below:
+
+### In CSS Files
+
+If your bundler (e.g., Vite) is configured to resolve npm packages in CSS imports:
+
+```css
+@import '@ks-no/designsystem-themes/base.css';
+@import '@ks-no/designsystem-themes/ledsagerbevis.css';
```

Comment:
Could we include base in custom themes to save consumers a line of imports?

Reply:
Themes are generated files so I think it is best to keep them separated